release/ray_release/cluster_manager/minimal.py

```diff
@@ -248,6 +251,39 @@ def create_cluster_compute(self, _repeat: bool = True):
                     f"ID {self.cluster_compute_id}"
                 )
 
+    @property
+    def cluster_compute_with_extra_tags(self) -> dict:
+        if self.extra_tags and self._get_cloud_provider() == "AWS":
```

Comment:
Instead of having one big if, maybe just return instead

```suggestion
        if not self.extra_tags or self._get_cloud_provider() != "AWS":
            return
```

this will make it better legible